tsconfig.json

```diff
@@ -11,7 +11,7 @@
         "emitDecoratorMetadata": true,
         "declaration": true,
         "declarationDir": "types",
-        "outDir": "tsc",
+        "outDir": "lib",
```

Comment:
This is a holdover from the previous approach but it also makes sense as this folder is transpiled, not TypeScript